docs/sources/index.md

```diff
@@ -293,5 +292,44 @@ TypesafeConfig.fromHoconString(jsonString, automaticDescription)
 
 
 ```
-
 Please check other ways to load the hocon file in `TypesafeConfig`
+
+## Command Line Arguments Source
+Simply source config from command line arguments
+```scala mdoc:silent
+final case class SimpleCommandLineConfig(key1: String, key2: String)
+
+val simpleCmdLineArgs = "--key1 value1 --key2 value2"
+val simpleSource = ConfigSource.fromCommandLineArgs(simpleCmdLineArgs.split(' ').toList)
+val simpleConfig = descriptor[SimpleCommandLineConfig] from simpleSource
+```
+With nested path in command line arguments
+```scala mdoc:silent
+final case class SparkConf(key1: String, key2: String)
+final case class NestedCommandLineConfig(conf: SparkConf, key3: String)
+
+val nestedCmdLineArgs = "--conf.key1 v1 --conf.key2 v2 --key3 v3"
+val nestedSource = ConfigSource.fromCommandLineArgs(nestedCmdLineArgs.split(' ').toList, keyDelimiter = Some('.'))
+val nestedConfig = descriptor[NestedCommandLineConfig] from nestedSource
+```
+The config with a complex source of command line arguments is what every production service needs
```

Comment:
Remove line number 315. Add the below heading. The heading is a subheading which is `###`
### Lists in Command Line Arguments: Approach 1

```
val listArgs = "--users Jane --users Jack"
val listSource = ConfigSource.fromCommandLineArgs(listArgs2.split(' ').toList
val config = list("users")(string) from listSource
assert( 
   read(config) == Right("Jane", "Jack")
)

```

Reply:
Add another heading
### Lists in Command Line Arguments (alternate approach)

```
val listArgs2 = "--users Jane,Jack"

// args.split(' ') is only for demo purpose. We already get a list if we use zio.App
val listSource2 = ConfigSource.fromCommandLineArgs(
   listArgs2.split(' ').toList,
   valueDelimiter = Some(',')
 )

list("user")(string) from listSource2 
```

Reply:
Please keep them as consistent as possible. I hope you find a pattern here